### PR TITLE
FOR-2437: Fixed textarea auto expanding issues with tabs component.

### PR DESCRIPTION
**Pull request by roflanKisel**

### CODE REVIEW

File: src/components/tabs/Tabs.js [hunk 1]
```diff
@@ -189,6 +181,16 @@ export default class TabsComponent extends NestedComponent {
       .addClass(this.tabLinks[index].tabLink, 'active')
       .addClass(this.tabs[index], 'active');
 
+    const components = this.hook('addComponents', tab.components, this);
```
Comment (travist) [inline src/components/tabs/Tabs.js]: Why was this needed? This seems pretty disruptive since I am not sure what this would do AFTER ```this.restoreValue()``` since I believe restoreValue should occur after the components have been added and not before. I think this will require a lot of testing before we can pull this in.

Comment (roflanKisel) [reply src/components/tabs/Tabs.js]: Moved it after adding components and tested it.
